test/src/main/java/org/apache/accumulo/test/metrics/fate/ZooLockTest.java

```diff
@@ -57,16 +57,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ZooLockIT extends SharedMiniClusterBase {
+public class ZooLockTest {
```

Comment:
While it may not need to extend SharedMiniClusterBase anymore, it is still an integration test, as it is specifically testing the integration between Accumulo code and a test instance of ZooKeeper that is started as an extra process. It should keep its original name, `ZooLockIT`.

The existing `FateMetricsTest` should probably also be called `FateMetricsIT` for the same reason, if you want to rename it for consistency, as part of this PR.

Reply:
This rule in our `pom.xml` will cause the build to fail if we keep the IT nomenclature and do not extend one of the classes (or add it to the test itself) that includes the import specified below: 
```
<classAnnotationPattern>org[.]junit[.]experimental[.]categories[.]Category</classAnnotationPattern>
```
https://github.com/apache/accumulo/blob/b12d1103d788473168616f28ae87a65bb76aa880/pom.xml#L1176-L1183

Reply:
I agree that this should be an IT. I was running into the issue that @Manno15 has outlined. I'm not sure what the best way to overcome this issue might be and am open to suggestions.

Reply:
The simplest solution is to just create a new annotation to use as a category of test. Perhaps `start/src/main/java/org/apache/accumulo/test/categories/ZooKeeperTests`

Reply:
@ctubbsii, the most recent commit should include the remaining changes you have mentioned.